cli/flox-rust-sdk/src/flox.rs

```diff
@@ -225,6 +225,30 @@ pub mod test_helpers {
     use super::*;
     use crate::providers::git::{GitCommandProvider, GitProvider};
 
+    pub fn create_test_token(handle: &str) -> Result<FloxhubToken, FloxhubTokenError> {
+        #[derive(Debug, Clone, Deserialize, Serialize)]
+        struct MyClaims {
+            #[serde(rename = "https://flox.dev/handle")]
+            handle: String,
+            exp: usize,
+        }
+        let my_claims = MyClaims {
+            handle: handle.to_owned(),
+            exp: 9999999999,
+        };
```

Comment:
suggestion; since this is just an adhoc json struct you can skip the type definition for 

```suggestion
        let my_claims = serde_json::json!({
            "https://flox.dev/handle": {handle},
            "exp": 9999999999,
        });
```